.github/workflows/release_v6.yml

```diff
@@ -0,0 +1,196 @@
+name: Release v6
```

Comment:
should adding this gh action be part of PR added to issue 👉 https://github.com/pnp/cli-microsoft365/issues/3435
TBH for me it wasn't that clear and I am a bit surprised it's here 😉.
maybe we could extend the issue description to align it with the PR?

Reply:
@martinlingstuyl sorry probably this is a really stupid question 😉. But do all v6 targeted PRs will push this v6 flow again  🤔. Shouldn't this end up there as part of the mergeing/rebeasing process 🤔?

Reply:
Hi @Adam-it , you're correct, they should not be part of this. I've now rebased my v6 PR's on the v6 branch. The v6 files should now be gone from my commits.

Reply:
The point was they were based on main. Which was why these commits from v6 were coming along